src/Core/src/Platform/Android/ImageButtonExtensions.cs

```diff
@@ -29,5 +29,22 @@ public static void UpdateCornerRadius(this ShapeableImageView nativeButton, IBut
 				.SetBottomRightCorner(CornerFamily.Rounded, buttonStroke.CornerRadius)
 				.Build();
 		}
+
+		public static void UpdatePadding(this ShapeableImageView nativeButton, IImageButton imageButton)
+		{
+			var context = nativeButton.Context;
+
+			if (context == null)
+			{
+				return;
+			}
+
+			nativeButton.SetContentPadding(		
+				(int)context.ToPixels(imageButton.Padding.Left),
+				(int)context.ToPixels(imageButton.Padding.Top),
+				(int)context.ToPixels(imageButton.Padding.Right),
+				(int)context.ToPixels(imageButton.Padding.Bottom)
+				);
```

Comment:
How does this work with default paddings and 0 or -1? I know there is a bunch of logic with the "default"/"normal" button. Most notably, we need to make sure the default button looks correct and a padding of 0 does not have a negative space and is in fact a 0 padding.